util/travis_compiled_push.sh

```diff
@@ -9,9 +9,10 @@ echo "Using git hash ${rev}"
 
 if [[ "$TRAVIS_BRANCH" == "master" && "$TRAVIS_PULL_REQUEST" == "false" ]] ; then
 
-# convert to unix line-endings
+# fix formatting
 git checkout master
 git diff --diff-filter=M --name-only -n 1 -z ${TRAVIS_COMMIT_RANGE} | xargs -0 dos2unix
+git diff --diff-filter=M --name-only -n 1 -z ${TRAVIS_COMMIT_RANGE} | grep -e '^drivers' -e '^quantum' -e '^tests' -e '^tmk_core' | xargs -0 clang-format
```

Comment:
I may be mistaken about when this will run, but near as I can tell this will only run clang-format on files that are modified, but not on new files that are added to the relevant directories, which would require `--diff-filter=AM` (added and modified).